index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nelson Araujo's Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    <header>
+        <section class="mobile-nav">
+            <div class="landing-header">
+                <div class="header-image"></div>
+                <button id='open-menu-btn' class="menu-btn">
+                    <img src="./assets/ic_menu.svg">
+                </button>
```

Comment:
For semantic reasons and to deliver a great user experience on mobile devices, especially for screen readers. I suggest you make the wrapper for this hamburger menu a `nav` tag